### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by matkrak**
Version no 1 - build passing in Travis CI.
Some things that should probably be done here:
- Find out which lowest versions of scipy and numpy can be used
- Add some lines to README.md - apt-get commands I used in .travis.yml are essential to get scipy and numpy
- fix setup.py so that install_requirements are not commended (and so that one could install it via pip+git)

### CODE REVIEW

**Comment by grzanka** [comment]
Put here only some dummy tests, rest will come with other pull request

File: README.md [hunk 1]
```diff
@@ -14,10 +15,25 @@ This module is dedicated for Python 3.x users
 Current version (0.1.0) is not available on PyPi, although once a <br>
 stable version is ready it will be pushed to PyPi repo.<br><br>
 
-For now, installation can be done from this GIT repository, using <br>
+
+As you may see in requirements.txt file beprof uses both numpy and scipy.<br>
+To make installing them possible, you might need to download some packages<br>
+which you can do (on UNIX) with:
```
Comment (grzanka) [inline README.md]: Does every UNIX have apt-get available ? What about RedHat, Slackware and pld ?

File: README.md [hunk 2]
```diff
@@ -14,10 +15,25 @@ This module is dedicated for Python 3.x users
...
+
+For now, installation can be done: 
+* via pip
+From this GIT repository, using <br>
 `~$ pip3 install git+https://github.com/grzanka/beprof.git@master` <br>
+(where `@master` refers to the name of a branch)<br>
 
-(where `@master` refers to the name of a branch)
+* manually
+Providing that you have already install necessery packages<br>
```
Comment (grzanka) [inline README.md]: Check grammar

File: README.md [hunk 3]
```diff
@@ -14,10 +15,25 @@ This module is dedicated for Python 3.x users
...
+```
+git clone https://github.com/grzanka/beprof.git
+cd beprof
+python setup.py install
```
Comment (grzanka) [inline README.md:35]: C:\Users\User\PycharmProjects\beprof>python setup.py install --user
C:\Users\User\Anaconda3\lib\distutils\dist.py:260: UserWarning: Unknown distribution option: 'install_requires'

I have numpy 1.9.4 and in my case install_requires is ignored.

Comment (matkrak) [inline README.md:35]: OK didn't try it on Windows. I will do it later

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+language: python
```
Comment (grzanka) [inline .travis.yml:1]: Add email notification

**Comment by grzanka** [comment]
Why travis displayes some Error1 and Error2 messages at the end of the log ?
https://s3.amazonaws.com/archive.travis-ci.org/jobs/110206426/log.txt

**Comment by matkrak** [comment]
This is because logging package wasnt merged yet, and change_domain prints error if it fails instead of throwing exception

File: README.md [hunk 5]
```diff
@@ -14,10 +15,25 @@ This module is dedicated for Python 3.x users
...
+
+For now, installation can be done: 
+* via pip
+From this GIT repository, using <br>
 `~$ pip3 install git+https://github.com/grzanka/beprof.git@master` <br>
+(where `@master` refers to the name of a branch)<br>
 
-(where `@master` refers to the name of a branch)
+* manually
+Providing that you have already installed necessery packages<br>
```
Comment (grzanka) [inline README.md:31]: More grammar to check - use spell checker

File: beprof/unittests.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        print('-'*10, 'PASSED', '-'*10)
+    else:
+        print('X'*10, 'FAILED', 'X'*10)
+
+    if pro.metadata['profile2'] == 'second_prof':
+        print('-'*10, 'PASSED', '-'*10)
+    else:
+        print('X'*10, 'FAILED', 'X'*10)
+
+if __name__=='__main__':
```
Comment (grzanka) [inline beprof/unittests.py:47]: Check how travis is evaluating whether build was successful or not. I suspect it checks return code. Typically 0 means success, others mean failure. Which return code will you get while running these tests ?

**Comment by grzanka** [comment]
Closing, as promised. In case of resuming activity, open new PR.

**Pull request closed without merging**